backend/pyclashbot/bot/navigation.py

```diff
@@ -297,6 +297,8 @@ def leave_end_battle_window(logger):
     if check_if_end_screen_is_exit_bottom_left():
         print("Leaving end battle (condition 1)")
         click(79, 625)
+        time.sleep(10)

```

Comment:
do the sleeps need to be this long or could they be shorter?

Reply:
We can make different sleep more good like this :
import pyautogui
import time

# Coordinates to click
x, y = 1, 2

# Click every second for 30 seconds
for i in range(30):
    pyautogui.click(x, y)
    time.sleep(1)